src/engine/HasRelationScan.h

```diff
@@ -0,0 +1,89 @@
+// Copyright 2018, University of Freiburg,
+// Chair of Algorithms and Data Structures.
+// Author: Florian Kramer (florian.kramer@mail.uni-freiburg.de)
+#pragma once
```

Comment:
I think we still also use header guards in the rest of the source

Reply:
Most of the operations seem to only use `#pragma once`